server_spec.rb

```diff
@@ -0,0 +1,234 @@
+require_relative 'server'
+require_relative 'spec_helper'
+
+describe Server do
+  let(:app) { described_class.new }
+  before :all do
+    @db = SQLite3::Database.new('test.db')
+  end
+
+  after :all do
+    File.delete('test.db')
+  end
+
+  def create_tables(db)
+    db.execute(<<-SQL
```

Comment:
Prefer to put complex setup code like this into helpers, rather than cluttering the test file.

Reply:
This would also seem to be common setup code that's relevant to the whole application - could it be extracted, and also made available as, say, a Rake task?